spring-batch-docs/asciidoc/step.adoc

```diff
@@ -249,15 +250,15 @@ were not declared to be abstract. The `Step`, "concreteStep2", has 'itemReader',
 ===== Merging Lists
 
 [role="xmlContent"]
-Some of the configurable elements on `Steps` are lists. The `<listeners/>` element, for
-instance. If both the parent and child `Steps` declare a `<listeners/>` element, then the
+Some of the configurable elements on `Steps` are lists, such as the `<listeners/>` element.
+If both the parent and child `Steps` declare a `<listeners/>` element, then the
 child's list overrides the parent's. In order to allow a child to add additional
 listeners to the list defined by the parent, every list element has a `merge` attribute.
 If the element specifies that `merge="true"`, then the child's list is combined with the
 parent's instead of overriding it.
 
 [role="xmlContent"]
-In the following example, the `Step`, "concreteStep3", is created with two listeners:
+In the following example, the `Step` "concreteStep3" is created with two listeners:
```

Comment:
I _think_ the coma after "concreteStep3" still belongs there...

Reply:
Yes, it does, because "concreteStep3" is parenthetical there. If you hate the commas, use parentheses. That may be more clear.